style.css

```diff
@@ -1,3 +1,361 @@
+  display: flex;
+  flex-direction: column;
+  padding: 1rem;
+}
+
+.nav-bar {
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+}
+  
+  /* .nav-bar {
+    background-color: lightskyblue;
+  width: 100%;
+  top: 0;
+} */
+
 #root {
-  color: red;
+  background-color: #8fe4df;
+  color: black;
+  padding: 1rem;
+}
+
+#search-input {
+  font-size: large;
+  text-align: center;
+}
+
+/* .episode-cards-container {
+  background-color: #E9E6E6;
+  border: 0.05rem #db9f9f solid;
+  border-radius: 0.5rem;
+  display: grid;
+  gap: 3rem;
+  justify-items: center;
+  justify-content: center;
+  padding: 2rem;
+  grid-template-columns: repeat(auto-fill, minmax(20rem, 1fr));
+} */
+
+.show-cards-container, .episode-cards-container {
+  background-color: #E9E6E6;
+  border: 0.05rem #db9f9f solid;
+  border-radius: 0.5rem;
+  display: grid;
+  gap: 3rem;
+  justify-items: center;
+  justify-content: center;
+  width: 18.25rem;
+  padding: 2rem;
+  grid-template-columns: repeat(auto-fill, minmax(20rem, 1fr));
+}
+
+.episode-card {
+  display: flex;
+  flex-direction: column;
+  border: 0.1rem black solid;
+  background-color: #D1EBF1;
+  width: 18.25rem;
+  border-radius: 0.3rem;
+  align-items: center;
+  padding: 1rem;
+}
+
+.show-name-text {
+  font-size: x-large;
+  font-weight: bold;
+  text-align: center;
+}
+
+.show-name-container {
+  grid-area: n;
+}
+
+.show-image-container {
+  display: flex;
+  flex-direction: column;
+  justify-self: center;
+  width: fit-content;
+  padding: 1rem;
+  background-color: #E9E6E6;
+  grid-area: i;
+}
+
+.show-summary-container {
+  background-color: #E9E6E6;
+  margin: 1rem;
+  padding: 1rem;
+  grid-area: s;
+}
+
+.show-summary-text {
+  width: 18rem;
+  text-align: justify;
+}
+
+
+.show-info-container {
+  display: flex;
+  flex-direction: column;
+  justify-self: center;
+  width: 18rem;
+  padding: 1rem;
+  background-color: #E9E6E6;
+  grid-area: b;
+}
+
+.episode-name-num-holder {
+  border: 0.1rem black solid;
+  border-radius: 0.3rem;
+  display: flex;
+  justify-content: center;
+  width: 20rem;
+}
+
+.episode-name-num-text {
+  font-family: "Lucida Sans", "Lucida Sans Regular", "Lucida Grande",
+    "Lucida Sans Unicode", Geneva, Verdana, sans-serif;
+}
+
+.episode-image-container {
+  padding: 1rem;
+}
+
+.episode-summary-text {
+  font-size: small;
+  padding: 0 1rem 1rem 1rem;
+  font-size: medium;
+}
+
+
+.show-card {
+  display: flex;
+  flex-direction: column;
+  border: 0.1rem #84d8ec solid;
+  background-color: #D1EBF1;
+  width: 18rem;
+  border-radius: 0.3rem;
+  justify-content: center;
+  /* align-items: center; */
+  padding: 1rem;
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  grid-template-rows: auto;
+  grid-gap: 2rem;
+  grid-template-areas:
+    "n"
+    "i"
+    "s"
+    "b";
+}
+
+
+#footer {
+  display: flex;
+  flex-direction: column;
+}
+
+#source {
+  display: flex;
+  justify-content: center;
+}
+
+
+@media screen and (min-width: 480px) {
```

Comment:
This is mobile-first approach, is that what is intended here?

Reply:
Yes, you are right. It is what I intended here.